sdk/translation/ai-translation-text/README.md

```diff
@@ -0,0 +1,324 @@
+
+Detect. Returns the source code language code and a boolean variable denoting whether the detected language is supported for text translation and transliteration.
+
+Dictionary lookup. Returns equivalent words for the source term in the target language.
+
+Dictionary example Returns grammatical structure and context examples for the source term and target term pair.
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/ai-translation-text)
+- [API reference documentation](https://learn.microsoft.com/azure/cognitive-services/translator/reference/v3-0-reference)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/translation/ai-translation-text/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- LTS versions of Node.js
+
+### Prerequisites
+
+- An existing Translator service or Cognitive Services resource.
+- Latest versions of Edge, Chrome, Safar and Firefox
+
+### Install the `@azure-rest/ai-translation-text` package
+
+Install the Azure Text Translation REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/ai-translation-text
+```
+
+#### Create a Translator service resource
+
+You can create Translator resource following [Create a Translator resource][translator_resource_create].
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+### Authenticate the client
+
+Interaction with the service using the client library begins with creating an instance of the [TextTranslationClient][translator_client_class] class. You will need an **API key** or `TokenCredential` to instantiate a client object. For more information regarding authenticating with cognitive services, see [Authenticate requests to Translator Service][translator_auth].
+
+#### Get an API key
+
+You can get the `endpoint`, `API key` and `Region` from the Cognitive Services resource or Translator service resource information in the [Azure Portal][azure_portal].
+
+Alternatively, use the [Azure CLI][azure_cli] snippet below to get the API key from the Translator service resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+### Create a `TextTranslationClient` using an API key and Region credential
+
+Once you have the value for the API key and Region, create an `TranslatorCredential`.
+
+With the value of the `TranslatorCredential` you can create the [TextTranslationClient][translator_client_class]:
+
+```typescript
+const translateCedential = new TranslatorCredential(apiKey, region);
+const translationClient = TextTranslationFactory(endpoint, translateCedential);
```

Comment:
Could we update all references to `TextTranslationFactory` to `TextTranslationClient`?

Reply:
Done